### PR TITLE
Update for .Net Core RTW.

### PR DESCRIPTION
**Pull request by dnduffy**

### CODE REVIEW

File: NuGet.config [hunk 1]
```diff
@@ -2,6 +2,9 @@
 <configuration>
   <packageSources>
     <add key="NuGet" value="https://api.nuget.org/v3/index.json" />
+    <!--<add key="AspNetVNext" value="https://www.myget.org/F/aspnetvnext/api/v3/index.json" />-->
+    <add key="XunitMyget" value="https://www.myget.org/F/xunit/api/v3/index.json" />
```
Comment (kartang) [inline NuGet.config]: Can we merge this to develop as opposed to master? Develop can later be merged to master, as we will not have myget feeds going out in "nuget.org" (which reflects the content in master).

File: test/Microsoft.ApplicationInsights.AspNetCore.Tests/project.json [hunk 2]
```diff
@@ -9,36 +9,37 @@
...
   },
   "testRunner": "xunit",
   "frameworks": {
     "netcoreapp1.0": {
       "imports": [
         "dnxcore50",
         "dotnet5.4",
-        "portable-net451+win8"
+        "portable-net451+win8",
+        "portable-dnxcore50+net45+win8+wp8+wpa81"
```
Comment (kartang) [inline test/Microsoft.ApplicationInsights.AspNetCore.Tests/project.json]: Is this required at all or accidentally left behind from the previous change?

File: src/Microsoft.ApplicationInsights.AspNetCore/project.json [hunk 3]
```diff
@@ -1,8 +1,8 @@
-﻿{
+{
   "title": "Application Insights for ASP.NET Core Web Applications",
   "description": "Application Insights for ASP.NET Core web applications. See https://azure.microsoft.com/en-us/documentation/articles/app-insights-asp-net-five/ for more information.",
   "authors": [ "Microsoft" ],
-  "version": "1.0.0-rc2-final",
+  "version": "1.0.0-rc3-final",
```
Comment (kartang) [inline src/Microsoft.ApplicationInsights.AspNetCore/project.json]: Can we change this to 1.0.0?

Comment (dnduffy) [inline src/Microsoft.ApplicationInsights.AspNetCore/project.json]: I did, I'm not sure how it got back to RC3... I'll fix it.

File: test/Microsoft.ApplicationInsights.AspNetCore.Tests/project.json (hunk 2 above)
Comment (dnduffy) [inline test/Microsoft.ApplicationInsights.AspNetCore.Tests/project.json]: Shouldn't be needed, I think it is left over from trying to get the RC3 stuff working.

File: NuGet.config (hunk 1 above)
Comment (dnduffy) [inline NuGet.config]: Yes, I've closed this PR and opened a new in against develop.

**Pull request closed without merging**